### PR TITLE
Reviewed and fixed docs for PortRanges.

### PR DESCRIPTION
**Pull request by mvidner**
Tests are still missing.
FlattenServices is a poorly understood beast.

### CODE REVIEW

**Comment by mvidner** [comment]
I did this as a helper in order to understand some code  in #457.

File: library/network/src/modules/PortRanges.rb [hunk 1]
```diff
@@ -33,24 +33,28 @@
 require "yast"
 
 module Yast
+  # Tools for ranges of network ports, as used by iptables for firewalling.
+  #
+  # A Port Range is a string of two numbers separated with a colon: "3000:3010".
+  # The range includes both ends. The numbers are nonnegative integers.
+  #
+  # A *Valid* Port Range is an ascenging pair of numbers between 1..65535.
```
Comment (imobach) [inline library/network/src/modules/PortRanges.rb]: ascen*d*ing?

**Comment by imobach** [comment]
Just a typo; LGTM otherwise.

**Comment by lslezak** [comment]
LGTM

**Pull request merged**